src/base.js

```diff
@@ -243,40 +243,20 @@ class Base {
       }
     }));
   }
+
   /**
-   * Send a message to the status room
-   *
-   * @param {object} options={} Optional options object: fixedWidthOutput:boolean
-   * @param {string} ...args additional arguments are formatted and send to the room
+   * Async call to get the status room ID
    *
-   * @returns {Promise}
+   * @params {_roomAliasLocalPart} Optional, the room alias local part
+   * @returns {Promise->string} Promise resolving the room ID
```

Comment:
is {Promise->string} actually correct syntax? I think I've done this in the past and it makes jsdoc throw